src/com/connectsdk/service/RokuService.java

```diff
@@ -702,18 +699,18 @@ public void onError(ServiceCommandError error) {
 
         String param;
         if (mimeType.contains("image")) {
-            param = String.format("15985?t=p&u=%s&h=%s&tr=crossfade",
-                    HttpMessage.encode(url), HttpMessage.encode(host));
+            param = String.format("15985?t=p&u=%s&tr=crossfade", HttpMessage.encode(url));
         } else if (mimeType.contains("video")) {
+            // parameter `h` is an event server URL
```

Comment:
I believe this comment is useless and probably misleading, because another person won't see this `h` in code.